bench/tx-generator/src/Cardano/Benchmarking/Compiler.hs

```diff
@@ -250,3 +260,46 @@ newWallet n = do
   name <- WalletName <$> newIdentifier n
   emit $ InitWallet name
   return name
+
+parseKey :: BSL.ByteString -> TextEnvelope
+parseKey x = case decode x of
+  Nothing -> error $ "parsing of key failed : " ++ show x
+  Just k -> k
+
+keyNameGenesisInputFund :: KeyName
+keyNameGenesisInputFund = KeyName "GenesisInputFund"
+
+keyNameTxGenFunds :: KeyName
+keyNameTxGenFunds = KeyName "TxGenFunds"
+
+-- addr_test1vzd3muund27y5nw83vymqj3a83pcuzkkejej6s75e5lfjcc85nc3p
+keyTxGenFunds :: TextEnvelope
+keyTxGenFunds = parseKey "{\n    \"type\": \"PaymentSigningKeyShelley_ed25519\",\n    \"description\": \"Payment Signing Key\",\n    \"cborHex\": \"5820617f846fc8b0e753bd51790de5f5a916de500175c6f5a0e27dde9da7879e1d35\"\n}\n"
+
```

Comment:
Why not define the values directly, like:
` = TextEnvelope "PaymentSigningKeyShelley_ed25519" "Payment Signing Key" "5820617f846fc8b0e753bd51790de5f5a916de500175c6f5a0e27dde9da7879e1d35"`

Reply:
That could be done.
This would look like:
```
    (TextEnvelope { teType = TextEnvelopeType "GenesisUTxOSigningKey_ed25519"
                  , teDescription = fromString "Genesis Initial UTxO Signing Key"
                 , teRawCBOR = "X \vl1~\182\201v(\152\250A\202\157h0\ETX\248h\153\171\SI/m\186\242D\228\NAK\182(&\162"})
```
One would have to convert die key to `RawCBOR`.
The way it is now one can just copy contents for the `.skey` file.

Reply:
The way it's done you still need some manual labour the `.skey` file (escaping the quotes, adding newlines).
You could just use the example I gave above (without field labels), extending the CBOR value to sth. like `(Base16.decodeLenient "5820617f846fc8b0e753bd51790de5f5a916de500175c6f5a0e27dde9da7879e1d35")`.
Every literal's type there has a `IsString` instance.
That way you'd just copy-paste the key file values, and not need to worry about inlining a JSON serialization.

Reply:
Yes indeet the `Base16` version looks nicer.